tests/extension/tests/index.js

```diff
@@ -74,13 +77,13 @@ describe('Chrome extension tests', function () {
 
   it('should allow worker.js to make outgoing call, and receive incoming call', async () => {
     const initButton = await page.$('#init');
+    await delay(1000); // allow time for page to render
```

Comment:
Is there an event that we can listen to when the page finished rendering? Adding an arbitrary delay may cause flakiness.

Reply:
Technically it should be finished rendering by line 79. I saw it flake once but that might have been from previous config. Let me remove this delay for now.